src/timeline/TimelineViewManager.cpp

```diff
@@ -13,6 +13,7 @@
 #include "MxcImageProvider.h"
 #include "UserSettingsPage.h"
 #include "dialogs/ImageOverlay.h"
+#include "../ui/UserProfile.h"
```

Comment:
`../` seems weird; it looks like `src` is in the include path by the looks of the other `#include`s?

Reply:
Sorry, I'll make the needed changes in the next commit !

Reply:
I don't think you need to apologize. This is WIP and it's alright to to have some rough edges and even then you probably don't need to apologize for your code. :3

Reply:
Sure, I wasn't apologizing exactly, forgive my poor english

Reply:
Ah, language barriers, makes sense. Didn't think of that :3